### DISCUSSION TITLE
Old and good deprecated packages

### DISCUSSION THREAD

**Discussion by dgorti** [vakata/jstree#2483] [General]
So when building the jstree, during mpm restore packages, npm compains about several packages  ~63 of them. It would be good to update these I would think. Could you please let us know if you are planning on this? just checking before I sink a bunch of time in this if needed.

**Comment by vakata** [comment]
The jstree v3 build procedure is very old indeed and the last time I updated it it took hours to fix. I will have a look later this week and let you know when I am done.

**Comment by vakata** [comment]
After updating grunt and all grunt plugins there are still a lot of deprecation warnings, but those are not direct dependencies of the build process, but rather dependencies of the used plugin. For now I will leave the build process as it is. Just to make sure that you know - those are only build process dependencies, jsTree v.3 itself only depends on jQuery.
